_episodes/02-practice-learning.md

```diff
@@ -75,61 +75,115 @@ tasks with normal effort under normal circumstances.
+because they draw upon an existing mental model to fill in another, speeding learning and making a memorable connection. However, all analogies have limitations!
+If you choose to use an analogy, be sure its usefulness outweighs its potential to generate misconceptions. 
+
+> ## Analogy Brainstorm
+>
+> 1. Think of an analogy to explore. Perhaps you have a favorite that relates to your area of professional interest, or a hobby. If 
+> you prefer to work with an example, consider this common analogy from education: "teaching is like gardening."
+> 2. Share your analogy with a partner or group. (If you have not yet done so, be sure to take a moment to introduce yourself, first!) What does your analogy
+> convey about the topic? How is it useful? In what ways is it wrong?
+> 
+>
+> This activity should take about 10 minutes.
+{: .challenge}
+
+> ## Analogies at Work: "Software Carpentry"
+> People often ask where our name came from. Greg Wilson has this to say:
+> 
+>"Brent Gorda and I came up with the name in 1998 to differentiate what we were teaching from software engineering. That's about digging the Channel Tunnel; 
+> we're about the computational equivalent of hanging drywall."
+> 
+> The word "carpentry" acts as a metaphor -- a type of analogy -- inspiring a comparison with something concrete,
+> hands on, practical, and useful. This clearly conveys the purpose of our organization: to support computational skill development
+> among working practitioners who need the right tools and practices to be effective day to day.
+{: .callout}
+
+A mental model may be represented as a collection of concepts and facts, connected by relationships. 
+The mental model of an expert in any given subject will be far larger and more complex than that of a novice, including both more concepts 
+and more detailed and numerous relationships. However, **both may be perfectly useful** in certain contexts. 
+
+Returning to our example levels of skill development: 
+*     A *novice* has a minimal mental model of surface features of the domain. Inaccuracies may interfere with adding new information.
+Predictions are likely to borrow heavily from mental models of other domains
 which seem superficially similar.
-*     A *competent practitioner* is someone who has a mental model that is good enough for everyday purposes. This model
-does not have to be completely accurate in order to be useful:
-for example, the average driver's mental model of how a car works probably does not include
-most of the complexities that a mechanical engineer would be concerned with.
+*     A *competent practitioner* has a mental model that is useful for everyday purposes. Most new information
+they are likely to encounter will fit well with their existing model. Even though many potential elements of their mental model may
+still be missing or wrong, predictions about their area of work are usually accurate.
```

Comment:
Is there a reason we don't describe the expert mental model briefly here? It seems odd to include it in the diagram but not mention it.

Reply:
I realize that we get to it later but I think it helps to define it here as well.